### DISCUSSION TITLE
Change precision of timestamps

### DISCUSSION THREAD

**Discussion by nome2050** [openai/whisper#1074] [Q&A]
Hey there is there any way i can get 4 decimal points of timestamps instead of two

eg: {'word': ' you.',
     'start': 288.0,
     'end': 288.32,
     'probability': 0.9976211190223694}]}],
 'language': 'en'}
 
 I need 'start': 288.0000 and end: 288.3242

**Comment by ddoron9** [comment]
I'm wondering the same issue. changing round 2 to 3 doesn't make any difference.

https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/timing.py#L328C1-L337C18

**Comment by toanhuynhnguyen** [comment]
You can try whisperx
